src/routes/route-help.tsx

```diff
@@ -39,8 +41,15 @@ export function RouteHelp() {
   }, [section]);
 
   return (
-    <div className="max-w-5xl bg-base rounded-lg px-6 mx-auto">
-      <Markdown>{content}</Markdown>
-    </div>
+    <>
+      <Breadcrumbs>
+        <BreadcrumbHome />
+        <Breadcrumb>Help</Breadcrumb>
```

Comment:
imo this should extract the title from the document somehow, maybe using front matter yaml. but that's a bit too complex to include in this PR

Reply:
or else we can switch to mdx, which would allow us to put the breadcrumbs react component into the markdown file itself, avoiding the need for any custom logic. That would work well unless we want to build a full fledged documentation feature with dozens of files